openai/assistant_manager.py

```diff
@@ -57,7 +57,10 @@ def run_assistant(self, instructions=""):
                 response_format=response_format
         ) as stream:
             stream.until_done()
-        return {"llm_response": event_handler.get_thread_messages(), "token_usage": event_handler.token_usage}
+        response = event_handler.get_response()
+        if response['status']:
+            return {"llm_response": response['message'], "token_usage": event_handler.token_usage}
+        raise Exception(response['message'])
```

Comment:
what does this exception means here?

Reply:
For unsuccessful run status, we are raising as exception instead of displaying bot message.